app/controllers/publishers/vacancies/documents_controller.rb

```diff
@@ -72,6 +74,25 @@ def after_document_delete_path
     end
   end
 
+  def send_dfe_analytics_event(event_type, name, size, content_type)
+    fail_safe do
+      event = DfE::Analytics::Event.new
+        .with_type(event_type)
+        .with_request_details(request)
+        .with_response_details(response)
+        .with_user(current_publisher)
+        .with_data(
+          vacancy_id: StringAnonymiser.new(vacancy.id),
```

Comment:
Not really sure if vacancy_id should be anonymised - Only need to do this if it is PII.